aws-lambda-java-serialization/src/main/resources/META-INF/services/com.amazonaws.services.lambda.serialization.CustomPojoSerializer

```diff
@@ -0,0 +1 @@
+com.amazonaws.services.lambda.serialization.JacksonPojoSerializer
```

Comment:
How will translate when GraalVM is applied? Will it still use reflection?

Reply:
From [1.0-RC8](https://www.graalvm.org/release-notes/release-candidates/#10-rc8):
> The native image generator now has automatic support for services loaded using ServiceLoader. All service implementation classes, listed in the META-INF directory, are available automatically as soon as the service interface is used. This eliminates the need to manually register resources and reflection support for such classes. The automatic registration can be disabled with the -H:-UseServiceLoaderFeature option.